pages/news/2020-07-09-intro-outro-sections.html

```diff
@@ -0,0 +1,106 @@
+title: Intro & Outro Cues, Silence Detection, and AutoDJ Improvements
```

Comment:
The other posts use this as a prefix for the title:

```suggestion
title: New in 2.3: intro & Outro Cues, Silence Detection, and AutoDJ Improvements
```

Reply:
done